main.py

```diff
@@ -72,7 +75,8 @@
         # Initialize DCG
         dcg = dcg_module.DCG(dcg_params)
         # Trains DCG and saves the model
-        dcg_module.train_DCG(dcg, dcg_params, train_loader, val_loader=val_loader)
+        dcg_module.train_DCG(dcg, dcg_params, train_loader,
```

Comment:
This feels  unnecessary. Why add a newline with no purpose?

Reply:
VS code extension added it. I tried to reverse it, it won't save without adding a line. Sorry